static/src/stylesheets/pasteup/_src.scss

```diff
@@ -1,2 +1,6 @@
+$browser-supports-columns: true !default;
+$browser-supports-flexbox: true !default;
```

Comment:
these are globals really, not unique to these modules. feel like they should be elevated out of individual components?